torch/_inductor/config.py

```diff
@@ -39,6 +39,17 @@ def bundle_triton_into_fx_graph_cache_default() -> Optional[bool]:
     )
 
 
+def prologue_fusion_enabled() -> bool:
```

Comment:
cc @c00w is there a way to do this with `Config` ? I see `justknob` arg but I don't see a way to compare the integer value of it to tell if its enabled or not

Reply:
This is all stylistic.

VERSION ints in JK are working around a missing feature in JK (enable based on build stamp). My suggestion is just use Config, and don't bother with versions. (If you want to turn something new on, just use a new JK, don't keep reusing the same one). However, as a matter of style other people have different preferences and do it differently.